samples/sensor/si_sensor/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  description: Silabs Si7006/13/20/21 temperature and humidity polling sample
+  name: si_sensor
```

Comment:
This sample should be made generic for more temperature and humidity sensors, no reason to limit it to Silabs sensors IMO. In any case, `si_sensor` is not a great name, since there exist other Silabs sensors than the RHT sensor.

Reply:
Ok. Apparently I'm stupid because a sample for this already exists at `samples/sensor/dht_polling`. I removed my sample